doc/apiv3.apib

```diff
@@ -157,12 +154,330 @@ he is allowed to see his own time entries in projects where the costs module is
+
+## Cost Entry [/api/v3/cost_entries/{id}]
+
++ Model
+    + Body
+
+            {
+                "_type": "CostEntry",
+                "_links": {
+                    "self": {
+                        "href": "/api/v3/cost_entries/1"
+                    },
+                    "project": {
+                        "href": "/api/v3/projects/1"
+                    },
+                    "costType": {
+                        "href": "/api/v3/cost_types/1"
+                    },
+                    "user": {
+                        "href": "/api/v3/users/1"
+                    },
+                    "workPackage": {
+                        "href": "/api/v3/work_packages/1"
+                    }
+                },
+                "id": 1,
+                "spentUnits": 3.14,
+                "spentOn": "2015-03-31",
+                "createdAt": "2015-03-31T08:51:20Z",
+                "updatedAt": "2015-03-31T08:51:20Z"
+            }
+
+
+## view Cost Entry [GET]
+
++ Parameters
+    + id (required, integer, `1`) ... Cost Entry id
+
++ Response 200 (application/hal+json)
+
+    [Cost Entry][]
+
++ Response 403 (application/hal+json)
+
+    Returned if the client does not have sufficient permissions.
+
+    **Required permission:** view cost entries **or** view own cost entries (on cost entry's project)
+
+    + Body
+
+            {
+                "_type": "Error",
+                "errorIdentifier": "urn:openproject-org:api:v3:errors:MissingPermission",
+                "message": "You are not allowed to see this cost entry."
+            }
+
+## Cost Entries by work package [/api/v3/work_packages/{id}/cost_entries]
+
++ Model
+    + Body
+
+            {
+                "_links":
+                {
+                    "self":
+                    {
+                        "href": "/api/v3/work_packages/1/cost_entries"
+                    }
+                },
+                "total": 1,
+                "count": 1,
+                "_type": "Collection",
+                "_embedded":
+                {
+                    "elements": [
+                        {
+                            "_type": "CostEntry",
+                            "_links": {
+                                "self": {
+                                    "href": "/api/v3/cost_entries/1"
+                                },
+                                "project": {
+                                    "href": "/api/v3/projects/1"
+                                },
+                                "costType": {
+                                    "href": "/api/v3/cost_types/1"
+                                },
+                                "user": {
+                                    "href": "/api/v3/users/1"
+                                },
+                                "workPackage": {
+                                    "href": "/api/v3/work_packages/1"
+                                }
+                            },
```

Comment:
You might want to mention the ``_embedded`` section in the example as it can add quite a bit to the response.